projects/backend/capi/articles.ts

```diff
@@ -293,90 +296,6 @@ const parseArticleResult = async (
     }
 }
 
-type CAPIEndpoint = 'preview' | 'printsent' | 'live'
```

Comment:
All the code below now moved to `article.ts` file inside the `util` folder so we can reuse it in `render.ts`